main.go

```diff
@@ -132,6 +148,14 @@ func main() {
 	}
 
 	server.HandlePublish = func(conn *rtmp.Conn) {
+		connectingKey := strings.ReplaceAll(conn.URL.Path, "/", "")
```

Comment:
Maybe we should only use the last element of the path as the connection key here?

e.g. `/live/publish/abc123` should probably still yield `abc123`.

Reply:
Also: I don't mind using `strings.ReplaceAll` here, we just need to bump the Go dep to 1.12 in the GitHub workflow.

Reply:
Yup totally agree.  Didn't think about that use case.  Tried a slightly different approach that should accept the key.  I didn't grab from the end trying to keep logic simple.  But can if it would be better